src/main/java/com/google/sps/tool/GoogleTranslate.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.tool;
+
+import com.google.cloud.translate.Translate;
+import com.google.cloud.translate.TranslateOptions;
+import com.google.cloud.translate.TranslateOptions.Builder;
+import com.google.cloud.translate.Translation;
+
+public class GoogleTranslate {
+
+  public static String translateText(String textNotTranslated, String targetLanguage) {
+    Translate translate = TranslateOptions.newBuilder().setProjectId("framecards").setQuotaProjectId("framecards").build().getService();
```

Comment:
can you pull this build/getService call out of this function? You probably don't want/need to rebuild it every time you translate some text.